www/src/examples/Form/InputSizes.js

```diff
@@ -1,7 +1,14 @@
-<>
+<Form.Group>
+  <Form.Label size="lg" col>
```

Comment:
can you fix this to show the actual API, and merge this in with the horizontal form groups section, or else make a new section paralleling the upstream TWBS docs?